### PR TITLE
Adding the Footer.

### PR DESCRIPTION
**Pull request by maciejmarchel12**
# Description

pr80

Please include a summary of the change and which issue is fixed using the headings below

## What

Changed the colour of the nav bar and footer for the page.

## Why

Changed was made according to comments on previous pr 

## How

Change is being made by using semantic ui colours 

## Verification Steps

What steps were taken to verify the change is working correctly

# Checklist:

- [ ] This pr is linked to the related trello card
- [x] My code follows the style guidelines of this project
- [x] I have requested a code review from (at least) two team members
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This change has been verified by another team member

### CODE REVIEW

File: pages/contact.html [hunk 1]
```diff
@@ -65,24 +66,72 @@
...
+            <div class="eight wide column">
+                <div class="ui link list">
+                <h4 class="ui header">Mount Sion C.B.S Waterford</h4>
+                <p>An all boys school located in the heart of Waterford City</p>
+                <p>It is the first ever school set up by Edmund Rice.</p>
+            </div>
+            </div>
+        </div>
+        <div class="ui section divider"></div>
+        <img class="ui centered small image" src="../assets/images/logo.jpg">
```
Comment (ciaranRoche) [inline pages/contact.html]: As requested in the previous PR either remove this logo or add a crest with no background. 
IMO having the school crest in the footer is a nice touch. Just with the white background, it does not look good removing the background would be a nice touch.

File: pages/contact.html [hunk 2]
```diff
@@ -65,24 +66,72 @@
...
+                        Department of Education
+                    </a>
+                    <a href="https://www.examinations.ie" class="item">
+                        State Exams Commission
+                    </a>
+                </div>
+            </div>
+            <div class="four wide column">
+                <div class="ui link list">
+                    <a href="http://www.pdst.ie" class="item">
```
Comment (ciaranRoche) [inline pages/contact.html]: Also as pointed out these are hard to read, this is because they are hyperlinks and are usually a bluish color by standard, if you go and try to override the `<a>` tag it will change for every hyperlink on the site, I would recommend maybe using some inline styling or else make a css class for them. I'll leave it up to you,

Comment (ciaranRoche) [reply pages/contact.html]: Awesome that works, good catch

**Comment by maciejmarchel12** [comment]
@ciaranRoche made changes according to comments.

File: pages/contact.html [hunk 3]
```diff
@@ -12,11 +13,11 @@
 </head>
 
 <body>
-
+    
     <div class="ui container">    
     <img class="ui centered medium image" src="../assets/images/logo.jpg">
 
-        <div class="stackable ui tabular menu">
+        <div class="inverted blue stackable ui tabular menu">
```
Comment (laurafitzgerald) [inline pages/contact.html]: I don't think this change should be in this pr as this one relates to the navbar.

**Comment by ciaranRoche** [comment]
@laurafitzgerald this was discussed in a previous PR, when it was requested to have the nav-bar stretched, it looked good on the guy's computers in the school but once you get to anything over 720 resolution the site looked all lopsided.
I agree on keeping the same, and at the moment since the navbar is merged we should follow suit with the size of the nav bar as to go and change the navbar again is a lot of work, and I feel it would be more beneficial to focus on other tasks.

**Comment by laurafitzgerald** [comment]
@ciaranRoche thanks for the info. I was missing some of the context there. From my pov the navbar looks good and wonder if we could have the footer the same width? When I view it on a wide screen it's very obvious that the footer and nav are different but they are the same colour so looks a bit funny.

File: pages/contact.html (hunk 3 above)
Comment (maciejmarchel12) [reply pages/contact.html]: @laurafitzgerald I changed it so it matched the footer on the page to see how it would look like.

File: pages/contact.html [hunk 4]
```diff
@@ -4,6 +4,7 @@
 <head>
     <title>Mount Sion CBS Secondary School</title>
 
+    <link rel="stylesheet" href="style.css">
```
[the remark this answers is not in the document]
Comment (maciejmarchel12) [reply pages/contact.html]: @laurafitzgerald I didn't add that, I think @dominikkawka might have added it from the last merge.

**Comment by dominikkawka** [comment]
You're going to need to rebase, we got a new semantics sheet. Just make sure that you save the footer in some file that won't get effected jic

**Comment by maciejmarchel12** [comment]
Well atm im trying to figure out how to down size the footer to match the nav bar width.

**Comment by maciejmarchel12** [comment]
@laurafitzgerald @ciaranRoche updated PR

**Comment by dominikkawka** [comment]
The mount sion CBS text at the bottom, I think that we should get rid of it. It takes up a bit of space and it doesn't look pretty

**Comment by dimitraz** [comment]
Just merged @dominikkawka's change to add the background image to the site. @maciejmarchel12 can you rebase please?

**Comment by judyobrienie** [comment]
@maciejmarchel12 Try this, it works locally for me
```
<div class="ui section divider"></div>
        <div class="ui horizontal inverted small divided link list">
            <a class="item" href="https://www.facebook.com/mountsioncbs"> <i class="large facebook square icon"></i></a>
            <a class="item" href="https://twitter.com/mountsion_cbs"><i class="large twitter square icon"></i></a>
            <p><sub>All rights reserved Mount Sion CBS</sub></p>
        </div>
```

**Comment by ciaranRoche** [comment]
@maciejmarchel12 try and rebase this off of master, as this works for both myself and @judyobrienie 

Also it could be worth checking against this [gist](https://gist.github.com/ciaranRoche/a2eabb1d617565db27544f496754bba8) its exactly what I have, your code may be missing something small.

icons

File: pages/contact.html [hunk 5]
```diff
@@ -85,8 +87,61 @@ <h1 class="ui dividing header" style="padding-top:8px">School Contact Details</h
...
+                </div>
+            </div>
+            <div class="eight wide column">
+                <div class="ui link list">
+                <h4 class="ui inverted header">Mount Sion C.B.S Waterford</h4>
+                <p>An all boys school located in the heart of Waterford City</p>
+                <p>It is the first ever school set up by Edmund Rice.</p>
+            </div>
+            </div>
+            <div class="ui horizontal inverted small link list">
```
Comment (ciaranRoche) [inline pages/contact.html]: This is in the wrong `div`, that is why it is not centered, you will need to move it out one you can do this by moving the closing `div` on line `143` to `136`.

It might be worth adding in a divider to break up the footer, please check against this [gist](https://gist.github.com/ciaranRoche/a2eabb1d617565db27544f496754bba8) and compare to what you got.

**Comment by ciaranRoche** [comment]
@maciejmarchel12 it appears your rebase is a commit behind, make sure you `git pull` from origin master before rebasing.

**Comment by maciejmarchel12** [comment]
@ciaranRoche for me it says my rebase isnt a commit behind, btw the background image wont work for me.

**Comment by maciejmarchel12** [comment]
@ciaranRoche also i got it centred but the icons wont show up either.

**Comment by maciejmarchel12** [comment]
Nvm we fixed the background image

**Comment by ciaranRoche** [comment]
@maciejmarchel12 out of interest, if you checkout `master` and run `git pull origin master` what is the output?

**Comment by maciejmarchel12** [comment]
@ciaranRoche 

wadaw

**Comment by ciaranRoche** [comment]
@maciejmarchel12 strange, after a rebase, the background should work out of the box, how did ye fix it?

**Comment by judyobrienie** [comment]
@maciejmarchel12 
git stash
git pull origin master

**Comment by maciejmarchel12** [comment]
@ciaranRoche just had to add ".." to <link rel="stylesheet" type="text/css" href="assets/css/style.css"> so it looks like <link rel="stylesheet" type="text/css" href="../assets/css/style.css"> thats how we fixed it.

**Comment by ciaranRoche** [comment]
@maciejmarchel12 those changes have been merged to master, you should not have to fix anything. If you checkout master, and then open it up in chrome is the background on all pages?

Also when you where rebasing, did you have conflicts you had to resolve?

**Comment by ciaranRoche** [comment]
@maciejmarchel12 yeah I rebased your PR off of master, it has unresolved conflicts, and I am guessing you pushed with force, so your current PR is now out of sync, your commits are on top of the second last commit.
If you look at this [gist](https://gist.github.com/ciaranRoche/ac2705239b4195173d1eebf38b40c269#file-rebase-sh-L7) you will see here the `CONFLICT` on `contacts.html`.
Do you know how work through and resolve these conflicts? Or do you need help with it?

**Comment by maciejmarchel12** [comment]
I fix the the conflicts in visual studio code and it works fine afterwards.

**Comment by ciaranRoche** [comment]
@maciejmarchel12 well done, can you push the changes, and we will try get an approval on this, and we close it out.

**Comment by maciejmarchel12** [comment]
@ciaranRoche pushed the changes

File: assets/css/style.css [hunk 6]
```diff
@@ -7,5 +7,5 @@ body {
 
 div.ui.divbg.container {
     background-color:white;
-    padding:1em;
+    padding:0.25em;
```
Comment (ciaranRoche) [inline assets/css/style.css]: Why was this changed?

File: pages/contact.html [hunk 7]
```diff
@@ -4,20 +4,23 @@
...
     <link rel="stylesheet" type="text/css" href="../assets/css/style.css">
-    <script src="https://code.jquery.com/jquery-3.1.1.min.js" integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
-        crossorigin="anonymous"></script>
+    <script
+      src="https://code.jquery.com/jquery-3.1.1.min.js"
+      integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8="
+      crossorigin="anonymous"></script>
     <script src="../semantic/out/semantic.min.js"></script>
-    <link rel="icon" href="../assets/favicon.ico" type="image/x-icon" />
     <script src="//code.jquery.com/jquery-2.2.1.js"></script>
```
Comment (ciaranRoche) [inline pages/contact.html]: this can be removed.

File: pages/contact.html [hunk 8]
```diff
@@ -4,20 +4,23 @@
...
 
 </head>
 
 <body>
 
     <div class="ui divbg container">
-        <img class="ui centered image" src="../assets/images/logo.jpg" height="128px">
+    <div class="ui container">
```
Comment (ciaranRoche) [inline pages/contact.html]: This can also be removed, and don't forget to remove the associated closing div tag.

File: pages/contact.html (hunk 5 above)
Comment (ciaranRoche) [reply pages/contact.html]: This still has to be fixed, check against the gist mentioned above.

File: pages/contact.html [hunk 9]
```diff
@@ -4,20 +4,23 @@
...
     <script src="//code.jquery.com/jquery-2.2.1.js"></script>
 
 </head>
 
 <body>
 
     <div class="ui divbg container">
-        <img class="ui centered image" src="../assets/images/logo.jpg" height="128px">
+    <div class="ui container">    
+    <img class="ui centered image" src="../assets/images/logo.jpg">
```
Comment (ciaranRoche) [inline pages/contact.html]: you will need to add `height="128px" to this

**Comment by laurafitzgerald** [comment]
just checking this out and can see that the social media icons are no longer centred. also the logo in the header is a lot bigger than on other pages now.

File: assets/css/style.css (hunk 6 above)
Comment (maciejmarchel12) [reply assets/css/style.css]: It was changed cause 1em was too wide and messed up the footer by adding a white background around it which was too big and made it look out of place.

Comment (ciaranRoche) [reply assets/css/style.css]: try changing this to `padding-top:1em;`

**Comment by maciejmarchel12** [comment]
Good news the icons fixed themselves.

**Comment by maciejmarchel12** [comment]
@ciaranRoche @laurafitzgerald @JameelB

File: pages/contact.html [hunk 10]
```diff
@@ -67,25 +70,77 @@
...
+            </div>
+            <div class="eight wide column">
+                <div class="ui link list">
+                <h4 class="ui inverted header">Mount Sion C.B.S Waterford</h4>
+                <p>An all boys school located in the heart of Waterford City</p>
+                <p>It is the first ever school set up by Edmund Rice.</p>
+                </div>
+            </div>
+        </div>
+            <div class="ui horizontal inverted small link list">
```
Comment (ciaranRoche) [inline pages/contact.html]: could you add the following just before this `div` : `<div class="ui section divider"></div>` it will add a small border line

**Comment by maciejmarchel12** [comment]
@ciaranRoche Added the break line as well as adding links to the icons.

File: pages/contact.html [hunk 11]
```diff
@@ -134,10 +134,12 @@ <h4 class="ui inverted header">Mount Sion C.B.S Waterford</h4>
                 </div>
             </div>
         </div>
+        <div class="ui section divider">
```
Comment (ciaranRoche) [inline pages/contact.html]: just a small thing, if you can move the closing `</div>` so that this looks like: `<div class="ui section divider"></div>` we do not want to wrap the list of social media links in a divider, we just want the divider to separate the two bodies of text

**Comment by ciaranRoche** [comment]
@maciejmarchel12 I am happy with this, ill hold off on approval till we get this on every page.

**Comment by maciejmarchel12** [comment]
@ciaranRoche So I can now add the footer to every page now?

**Comment by maciejmarchel12** [comment]
@laurafitzgerald @judyobrienie @dimitraz @JameelB waiting for more comments so I can start adding it to other pages

**Comment by JameelB** [comment]
@maciejmarchel12 @ciaranRoche Looks good to me :+1: Feel free to add it to every page. I'll approve once all pages has the footer.

**Comment by maciejmarchel12** [comment]
I changed the text in the footer to ```
<p>An all boys school located in the heart of Waterford City</p>
<p>It is the founding school set up by Edmund Rice.</p>
```
which imo would be better than saying it was the first school, request was made by Mr.Galvin

**Comment by maciejmarchel12** [comment]
@judyobrienie It wont let me change it to be completely white, the closest I can get it to be white is if I invert the link list.

**Comment by maciejmarchel12** [comment]
@ciaranRoche @laurafitzgerald @dimitraz @judyobrienie @JameelB added the footer to all pages

**Comment by judyobrienie** [comment]
@maciejmarchel12 Excellent work, well done.

**Comment by maciejmarchel12** [comment]
@ciaranRoche @laurafitzgerald @JameelB  @dimitraz @judyobrienie Rebased

**Comment by dimitraz** [comment]
Looks good, just extra icons in the contact.html footer

**Comment by maciejmarchel12** [comment]
Removed the extra icons
